### PR TITLE
Update Helm Chart with v2.5.0

### PR DESCRIPTION
**Pull request by Starefossen**
This Pull Request updates the Helm Chart to the v2.5.0 version of the dashboard.

### CODE REVIEW

**Comment by floreks** [comment]
I think for the release we are bumping the minor version of the chart, right @desaintmartin?

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.1.3
-appVersion: 2.4.0
+version: 5.1.4
```
Comment (Starefossen) [inline aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml:17]:
```suggestion
version: 5.2.0
```

**Comment by desaintmartin** [comment]
It's a matter of style, but, indeed, it's cleaner if [bump of minor version of dashboard] -> [bump of minor version of the chart].

**Review by desaintmartin — APPROVED** [review]

**Comment by desaintmartin** [comment]
/lgtm
Thanks!

**Pull request merged**